src/tests/test_cfd_verifier.cpp

```diff
@@ -0,0 +1,77 @@
+#include <iostream>
+#include <string>
+#include <vector>
+
+#include <gtest/gtest.h>
+
+#include "algo_factory.h"
+#include "algorithms/cfd/cfd_verifier/cfd_verifier.h"
+#include "all_csv_configs.h"
+#include "config/names_and_descriptions.h"
+
+namespace tests {
+
+struct CFDVerifierParams {
+    algos::StdParamsMap params;
+    bool expect_holds;
+
+    CFDVerifierParams(std::vector<std::pair<std::string, std::string>> left,
+                      std::pair<std::string, std::string> right,
```

Comment:
you include `#include "algorithms/cfd/cfd_verifier/cfd_verifier.h"`, which has
```cpp
using CFDAttributeValuePair = std::pair<std::string, std::string>;
```
consider using it here